src/main/java/com/marklogic/mule/extension/api/ExecuteErrorsProvider.java

```diff
@@ -0,0 +1,17 @@
+package com.marklogic.mule.extension.api;
+
+import org.mule.runtime.extension.api.annotation.error.ErrorTypeProvider;
+import org.mule.runtime.extension.api.error.ErrorTypeDefinition;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class ExecuteErrorsProvider implements ErrorTypeProvider {
```

Comment:
Toss in a comment with a link to the Mule docs page that mentions this, as this concept seems important to have a reference to.